libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -1260,5 +1253,60 @@
         "value": "Remove from selection.",
         "comment": "Text of the tooltip shown above button that removes user or group from unsaved selection of newly added grantees that will gain access right to the dashboard after changes in share dialog are saved.",
         "limit": 0
+    },
+    "sorting.dropdown.header": {
+        "value": "Sort insight",
+        "comment": "",
```

Comment:
Comments for translation agency are missing for all the texts. Shouldn't there be some context?

Reply:
Added.